### DISCUSSION TITLE
Dockerhub / ECR images over a year out of date?

### DISCUSSION THREAD

**Discussion by Kilowhisky** [developmentseed/titiler#630] [Q&A] [answered]
Last hub images are over a year out of date. Should they be updated or is this method deprecated? 

https://hub.docker.com/r/developmentseed/titiler/tags

**Comment by vincentsarago** [comment, marked as answer]
we don't publish docker image on ECR and Dockerhub anymore 

see https://github.com/developmentseed/titiler/pkgs/container/titiler

**Reply by Kilowhisky** [reply]
So pull them or add a note?

**Reply by vincentsarago** [reply]
added a note!
